src/WCSimTuningParameters.cc

```diff
@@ -19,6 +19,9 @@ WCSimTuningParameters::WCSimTuningParameters()
  tvspacing = 100.0;
  topveto = false;
 
+ WCCladdingReflectivity   = 0.90; //
+ WCODTyvekReflectivity   = 0.97; //
```

Comment:
```suggestion
 WCODTyvekReflectivity   = 0.90; //
```
the default appears to be 0.97 in WCSimTuningMessenger.cc and tuning_parameters.mac

Reply:
Good spot! yes, I will implement this suggestion.